backend/requirements/base.in

```diff
@@ -17,6 +17,7 @@ django-solo
 mozilla-django-oidc-db[setup-configuration]
 django-privates
 django-setup-configuration
+django-health-checks @ git+https://github.com/maykinmedia/django-health-checks.git@1be9adb70cea0277e0eb00d42a7d751d70910f45
```

Comment:
Is there any reason to use the GH dependency link, if it's temperory untill something lands I think it's OK to use this, but efforts are made to restrict python dependencies to pypi only.

Can we either:

 - Use an upstream library
 - Create an issue (and comment here) to update eventuelly